tests/acceptance/test_acceptance.py

```diff
@@ -22,6 +29,12 @@ def test_robot():
     pytest.importorskip("JupyterLibrary")
 
     env = dict(**os.environ)
+    robot_args = ["robot", "--consolecolors=on", f"--outputdir={OUTPUT}"]
+
+    if NOTEBOOK_VERSION >= 7:
```

Comment:
This variable could be None, then we would run into `TypeError: '>=' not supported between instances of 'NoneType' and 'int'`.

```suggestion
    if NOTEBOOK_VERSION and NOTEBOOK_VERSION >= 7:
```

Reply:
8f0ff5e

Turns out we _could_ also test the classic stuff by using the `nbclassic` shim, but... meh.